desc/geometry/curve.py

```diff
@@ -293,15 +299,17 @@ def from_values(cls, coords, N=10, NFP=1, sym=False, basis="rpz", name=""):
         )
 
         grid = LinearGrid(zeta=phi, NFP=1, sym=sym)
-        basis = FourierSeries(N=N, NFP=NFP, sym=sym)
-        transform = Transform(grid, basis, build_pinv=True)
-        R_n = transform.fit(R)
-        Z_n = transform.fit(Z)
+        R_basis = FourierSeries(N=N, NFP=NFP, sym="cos" if sym else False)
```

Comment:
was this incorrect before then?

Reply:
Yes I think it was. `FourierSeries` has no `sym=True` option, so we probably just never tested that case.